cartographer_ros_msgs/msg/metrics/HistogramBucket.msg

```diff
@@ -0,0 +1,19 @@
+# 2018 The Cartographer Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# A histogram bucket between two boundaries i,j counts values x for which:
+#   bucket_boundary_i < x <= bucket_boundary_j
+# holds.
+float64 bucket_boundary
```

Comment:
@gaschler PTAL

Reply:
Replace i, j by "this" boundary and "previous" or the like